flexmeasures/data/schemas/forecasting/pipeline.py

```diff
@@ -236,7 +243,7 @@ def resolve_config(self, data: dict, **kwargs) -> dict:  # noqa: C901
             target=target_sensor,
             model_save_dir=data["model_save_dir"],
             output_path=output_path,
-            start_date=data["start_date"],
+            start_date=start_date,
```

Comment:
Likewise, doesn't this crash on an undefined `start_date` variable, if `data["start_date"] is not None`?